src/renderer/account/components/Management/EncryptedInput/EncryptedInput.js

```diff
@@ -71,7 +71,7 @@ export default class EncryptedInput extends React.PureComponent {
 
     try {
       if (wallet) {
-        const walletInstance = Wallet({ encryptedMnemonic: data, passphrase, wallet });
+        const walletInstance = await Wallet({ encryptedMnemonic: data, passphrase, wallet });
```

Comment:
Would be nice if this was an actual class, but not relevant to this PR

Reply:
you cannot `await new Wallet()`, right :P That's what I meant with refactoring it to be sync